src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1079,11 +1079,11 @@ private GencodeFuncotation createExonFuncotation(final VariantContext variant,
 
         // Before we get started, check to see if this is a non-protein-coding feature.
         // If it is, we must handle it differently:
-        if ( transcript.getGeneType() != GencodeGtfFeature.GeneTranscriptType.PROTEIN_CODING) {
-            return createCodingRegionFuncotationForNonProteinCodingFeature(variant, altAllele, reference, transcript, exon);
+        if ( transcript.getGeneType().equals(GencodeGtfFeature.KnownGeneBiotype.PROTEIN_CODING.toString()) ) {
```

Comment:
Can `transcript.getGeneType()` be null? If so, reverse the order of the comparison (comment also applies to code below)

Reply:
👍